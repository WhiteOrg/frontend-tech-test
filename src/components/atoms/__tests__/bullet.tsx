import React from "react";
import renderer from "react-test-renderer";
import Bullet from "../bullet";

describe("ATOMS: Render testing", () => {
  test("Should render Bullet", () => {
    const component = renderer.create(
      <ul>
        <Bullet>
          Advertising is key factor in your web site promotion. Let people talk
          about you, your products, and your services.
        </Bullet>
        <Bullet>
          You see, people are always on the look out for good content right?
          Your site should have really good contents. Don’t write
        </Bullet>
        <Bullet>
          “Welcome to my website” or “Ours is a World’s first company…blah blah”
          unless and until you really have dominating product.{" "}
        </Bullet>
        <Bullet>
          You sell your products or services as like others, but you think you
          must be noticed.{" "}
        </Bullet>
        <Bullet>
          Your offer should be different to attract visitor. Your presentation
          should be mind catching.{" "}
        </Bullet>
        <Bullet>
          Get aware of advertisers online. There are few major players around.{" "}
        </Bullet>
      </ul>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
