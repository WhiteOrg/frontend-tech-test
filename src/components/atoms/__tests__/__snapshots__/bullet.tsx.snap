// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATOMS: Render testing Should render Bullet 1`] = `
<ul>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Advertising is key factor in your web site promotion. Let people talk about you, your products, and your services.
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    You see, people are always on the look out for good content right? Your site should have really good contents. Don’t write
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    “Welcome to my website” or “Ours is a World’s first company…blah blah” unless and until you really have dominating product.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    You sell your products or services as like others, but you think you must be noticed.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Your offer should be different to attract visitor. Your presentation should be mind catching.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Get aware of advertisers online. There are few major players around.
     
  </li>
</ul>
`;

exports[`ATOMS: Render testing Should render bullet 1`] = `
<ul>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Advertising is key factor in your web site promotion. Let people talk about you, your products, and your services.
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    You see, people are always on the look out for good content right? Your site should have really good contents. Don’t write
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    “Welcome to my website” or “Ours is a World’s first company…blah blah” unless and until you really have dominating product.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    You sell your products or services as like others, but you think you must be noticed.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Your offer should be different to attract visitor. Your presentation should be mind catching.
     
  </li>
  <li
    className="sc-AxjAm lnHWwf"
  >
    Get aware of advertisers online. There are few major players around.
     
  </li>
</ul>
`;
